src/Turbulence_PrognosticTKE.jl

```diff
@@ -925,17 +918,8 @@ function GMV_third_m(
         @. ϕ_gm += a_up * Ic(var_up)
     end
 
-    if is_tke
-        parent(ϕ_en_cov) .= 0
-        @inbounds for i in 1:N_up
-            horiz_K_eddy = aux_up_c[i].horiz_K_eddy
-            a_up = aux_up_c[i].area
-            a_bulk = aux_bulk.area
-            @. ϕ_en_cov += -horiz_K_eddy * ∇c(wvec(w_en)) * a_up / a_bulk
-        end
-    else
-        @. ϕ_en_cov = covar_en
-    end
+    # w'w' ≈ 2/3 TKE (isotropic turbulence assumption)
+    @. ϕ_en_cov = is_tke ? 2.0 / 3.0 * covar_en : covar_en
```

Comment:
```suggestion
    @. ϕ_en_cov = is_tke ? FT(2 / 3) * covar_en : covar_en
```
and call `FT = eltype(grid)` above. I'm a bit impressed this works. Maybe take a look at the allocations analysis to make sure this doesn't pop up as a major contributor. If it does, I'd just change this to
```julia
    @. ϕ_en_cov = is_tke ? FT(2 / 3) * covar_en : covar_en
    if is_tke
        @. ϕ_en_cov = FT(2 / 3) * covar_en
    else
        @. ϕ_en_cov = covar_en
    end
```